### ISSUE TITLE
Single Laptop

### ISSUE THREAD

**Issue by emari2** [ossec/ossec-hids#814]
Is it easy and worth to install ossec in a single laptop or it is too much?

**Comment by aremai** [comment]
Hi @emari2 ,

hmm, well if the operating system of your laptop is a windows, then I'm afraid it's impossible to run OSSEC standalone. The standalone (local installation) of OSSEC requires a Linux (UNIX-based) operating system, where you can register also Windows agents (workstations). 
It does make sense to also run OSSEC on laptops/workstations, a lot of companies are required to due to compliance reasons.

bottom line: If your laptop runs linux as the operating systems, then you should be fine with the local installation :) there's even pre-compiled binaries (debian packages for debian/ubuntu and rpm packages for red hat/centos/fedora) available :)

**Comment by emari2** [comment]
Thanks a lot. No. I am not talking about windows I just was trading your pages in windows
I am instaling things in Debían jessie. A ppreciate your información about the binaries. My questions are because I an new to Linux, well new in programing but I have using ubuntu for a while but not the terminal. NowvI want steps ahead and I will installed it in Debían and hope you help me in get it up and runing, just in case
I already installed Ufw, clamav, clamtk and LMD.
My NeXt step is ossec.
Regards
Emilio

El 28 de abril de 2016 13:41:42 CEST, Theresa Meiksner <notifications@github.com> escribió:
>Hi @emari2 ,
>
>hmm, well if the operating system of your laptop is a windows, then I'm
>afraid it's impossible to run OSSEC standalone. The standalone (local
>installation) of OSSEC requires a Linux (UNIX-based) operating system,
>where you can register also Windows agents (workstations). 
>It does make sense to also run OSSEC on laptops/workstations, a lot of
>companies are required to due to compliance reasons.
>
>bottom line: If your laptop runs linux as the operating systems, then
>you should be fine with the local installation :) there's even
>pre-compiled binaries (debian packages for debian/ubuntu and rpm
>packages for red hat/centos/fedora) available :)
>
>---
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/ossec/ossec-hids/issues/814#issuecomment-215398446

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by emari2** [comment]
What should I install in my laptop?
Ossec-hids
Or
Ossec-hids-agent?

**Comment by aremai** [comment]
hi @emari2 , you should be fine with installing ossec-hids
the agent requires an ossec-manager (server) to report to... you probably won't have this when you just want to install it on your laptop.

so my recommendation would be: install it on your laptop, play around with it, and if you like expand it in a bigger environment, where you will also have a ossec-manager.

of course you could just follow the instructions on this page
[http://ossec.github.io/downloads.html](url)

this would install the manager, which can be run as a "local" installation, when no agents are connected.

hope this helps.

**Comment by emari2** [comment]
Thanks a lot. I will do that and I am sure I will have mire questions. Thanks again to be HERE for help. Nice weekend
